data/entities.yaml

```
@@ -1193,3 +1193,33 @@
     inner workings.
   properties: [portable]
   capabilities: [debug]
+
+- name: victrola
+  display:
+    attr: device
+    char: 'Q'
+  description:
+  - |
+    A device for reading and writing data on circular platters made of
+    a soft plastic material.  The stylus must be made of an extremely
+    hard and durable material, with a special tip that relies on
+    quantum effects to extract information stored at very high density.
```

There's a bit of "superlative overload" here.  How about:

```suggestion
    A device for reading and writing data on circular platters made of
    a soft plastic material.  The stylus must be made of a
    hard and durable material, with a special tip that relies on
    quantum effects to extract high-density information.
```